core/src/main/java/io/grpc/internal/AbstractStream2.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2014, Google Inc. All rights reserved.
```

Comment:
Believe it or not, this started as copy of `AbstractStream.java`. The only reason it is a new file is to avoid updating client-side at the same time. I'd just assume leave the year alone.